reducers/storage.ts

```diff
@@ -110,6 +111,49 @@ function storage(state: Record<string, any> = {}, action: GenericAction) {
     }
 }
 
+function migrateDrafts(state: any) {
+    const drafts: any = {};
+    for (const storageKey of Object.keys(state)) {
+        if (!storageKey.startsWith('draft')) {
+            continue;
+        }
+
+        const storageDraft = state[storageKey];
+        if (storageDraft.value?.channelId) {
+            // No migration is needed
+            continue;
+        }
+
+        const info = getDraftInfoFromKey(storageKey, StoragePrefixes.DRAFT);
+        const timestamp = new Date(storageDraft.timestamp);
```

Comment:
why do we create a new date timestamp her? Can't we just use `storageDraft.timestamp` directly?

Reply:
It was not recognizing `storageDraft.timestamp` as a `Date` so this was necessary to do any operations on it.